### PR TITLE
Register QDates [https://github.com/antimon2/QDates.jl.git]: v0.0.1

### PR DESCRIPTION
**Pull request by antimon2**
I tried `PkgDev.publish()` but an Error occured.  
So I try to send PR manually.

---

Error:

```
julia> PkgDev.publish()
INFO: Validating METADATA
ERROR: packages with unsatisfiable requirements found:
    EEG v0.0.3 – no valid versions exist for package ImageFiltering

 in check_metadata(::Set{String}) at /path/to/home/.julia/v0.5/PkgDev/src/entry.jl:301
…<omitted>
```

### CODE REVIEW

File: QDates/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.5
+Compat
```
Comment (tkelman) [inline QDates/versions/0.0.1/requires]: It's best to declare a minimum version dependency that implements the features you need.

**Comment by antimon2** [comment]
@tkelman Set the minimum version of `Compat`.

**Comment by antimon2** [comment]
I abandon this  PR once.
I will PR again later.

File: QDates/versions/0.0.1/requires [hunk 2]
```diff
@@ -1,2 +1,2 @@
 julia 0.5
-Compat
+Compat 0.11
```
Comment (tkelman) [inline QDates/versions/0.0.1/requires]: what feature are you relying on?

**Pull request closed without merging**